DOC-7523 -- Remove Inter-sync-gateway v1 and v2 references

ibsoln opened the pull request:
https://issues.couchbase.com/browse/DOC-7523

Interim -- clean-up all varieties of v1 mentions to be SG Replicate
Interim -- Removed v2 references

ibsoln committed:
DOC-7523 -- Remove Inter-sync-gateway v1 and v2 references
https://issues.couchbase.com/browse/DOC-7523

Interim -- clean-up all varieties of v1 mentions to be SG Replicate
Interim -- Removed v2 references

rajagp commented on modules/ROOT/pages/_partials/block-authors-notes.adoc:
```diff
@@ -39,7 +39,7 @@ ifeval::["{do-it}" == "yes"]
 CONTENT UNDER DEVELOPMENT
 
-This content relates to ICR on pre-2.8 deployments using Inter-Sync Gateway Replication (v1) protocol. It is in the process of being updated to reflect the state in release 2.8.
+This content relates to ICR on pre-2.8 deployments using SG{nbsp}Replicate protocol. It is in the process of being updated to reflect the state in release 2.8.
```
Global: Please avoid use of "ICR" (or non-industry acronyms)

rajagp replied:
Global: Not sure if this "Authors notes" is a common thing across all our platforms or industry practice..just seems like the doc is beta quality .Why are we publishing work thats in progress

ibsoln replied:
It was used as an include on under-development pages. It's no longer required now and I will remove.

rajagp commented on modules/ROOT/pages/_partials/block-authors-notes.adoc:
```diff
@@ -39,7 +39,7 @@ ifeval::["{do-it}" == "yes"]
 ====
 CONTENT UNDER DEVELOPMENT
```
General: Not sure of the purpose of the "Content Under Development" tag. Not sure if its common across server etc. Seems very odd to have that in official non-beta docs. Please clean up. Docs will always evolve and technically will always remain under development

ibsoln replied:
as above

ibsoln replied:
Content in _partials is used in includes. This is just a utility 'page' containing blocks to be used on a page as and when needed; a macro if you like that helps ensure consistency of presentation.

rajagp commented on modules/ROOT/pages/_partials/block-caveats.adoc:
```diff
@@ -1,7 +1,7 @@
 
 // tag::icr-community-redirect[]
-Unless otherwise stated this content relates only to replications run using the Inter-Sync Gateway Replication (v2) protocol in {enterprise} deployments.
+Unless otherwise stated this content relates only to Inter-Sync Gateway replications in {enterprise} deployments.
```
I do not know the context of this statement and page. Sounds confusing.

In general community is a strict subset of enterprise features. And we should not be confusing "Enterprise deployments" with "Enterprise Edition" .... Community edition can be deployed by Enterprises.

ibsoln replied:
{enterprise} expands as Enterprise Edition with a link to the comparison page https://www.couchbase.com/products/editions

rajagp commented on modules/ROOT/pages/_partials/block-caveats.adoc:
```diff
@@ -37,7 +37,7 @@ For documentation on later Inter-Sync Gateway Replication features see {xref-sgw
 [CAUTION]
 --
-This content relates only to replications run using the latest version of Inter-Sync Gateway Replication (release 2.8+).
+This content relates only to Inter-Sync Gateway replication.
```
```suggestion
This content relates only to Inter-Sync Gateway replications using the version of Sync Gateway 2.8+.
```
GENERAL : Don't treat "Inter- Sync Gateway replication" to be synonymous to what's introduced in 2.8. Treat it as a capability

rajagp replied:
GENERAL: Unless in headings, please do not capitalize "I" or "R" in "Inter-Sync Gateway Replication"...others seems like a title/name..so if its used in middle, you should just say "inter-Sync Gateway replication" (only the proper noun is capitalized...this is what we are doing on our couchbase web pages)

rajagp replied:
GENERAL: I am not sure if we need to keep calling out "Inter-SGW replications" every where in a page which is only  about that capability , so we should just use "replications" where it makes sense. Otherwise, it seems like we are "naming" this thing..which we are not. 
Eg "Replicating using "Inter-Sync Gateway Replication"" or "INtializing Inter-Sync Gateway replication" ...The first example does not make much sense as this page is about replication but the second example should say "Initialization: 

 Its similar to "peer-to-peer" replications - we don't keep stating p2p everywhere in that page. We just refer to that as "replication" and in context is the page, it makes sense that its "peer-to-peer"

ibsoln replied:
Ok, that makes sense

rajagp commented on modules/ROOT/pages/_partials/common-releasenotes.adoc:
```diff
@@ -93,13 +93,13 @@ include::{module-partials}pn-issues-list.adoc[tag=deprecated, leveloffset=3]
-===== Minimum Version for Inter-Sync Gateway Replication (v2)
-In order to support Inter-Sync Gateway Replication (v2), all active nodes in active cluster must be running Sync Gateway 2.8+.
+===== Minimum Version for Inter-Sync Gateway Replication
+In order to support Inter-Sync Gateway Replication, all active nodes in active cluster must be running Sync Gateway 2.8+.
```
```suggestion
In order to support the version of Inter-Sync Gateway Replication introduced with 2.8, all nodes in the active cluster must be running Sync Gateway 2.8+.
```

rajagp commented on modules/ROOT/pages/_partials/common-releasenotes.adoc:
```diff
@@ -123,7 +123,7 @@ However, root level replications (which by definition, use Inter-Sync Gateway Re
 
 // tag::upgrading[]
-In order to support Inter-Sync Gateway Replication (v2), all active nodes in an active cluster must be running Sync Gateway version 2.8+.
+In order to support Inter-Sync Gateway Replication, all active nodes in an active cluster must be running Sync Gateway version 2.8+.
```
```suggestion
In order to support the version of Inter-Sync Gateway Replication introduced with 2.8, all nodes in the active cluster must be running Sync Gateway 2.8+.
```

rajagp commented on modules/ROOT/pages/_partials/common-releasenotes.adoc:
```diff
@@ -132,13 +132,13 @@ The version of Inter-Sync{nbsp}Gateway useable depends on the Sync Gateway versi
-* Pre-Sync Gateway 2.8 -- only Inter-Sync Gateway Replication version 1 is available
-* For Pre-Sync Gateway 2.8+ -- you may run Inter-Sync Gateway Replication version 2 and Inter-Sync Gateway Replication version 1 (deprecated)
+* Pre-Sync Gateway 2.8 -- only SG{nbsp}Replicate is available
+* For Pre-Sync Gateway 2.8+ -- you may run Inter-Sync Gateway Replication and SG{nbsp}Replicate (deprecated)
```
The second line is confusing. "Pre-Sync Gateway 2.8+" ?? Did you mean "Sync Gateway 2.8" ?

ibsoln replied:
indeed I did

rajagp commented on modules/ROOT/pages/_partials/common-releasenotes.adoc:
```diff
@@ -132,13 +132,13 @@ The version of Inter-Sync{nbsp}Gateway useable depends on the Sync Gateway versi
-* Pre-Sync Gateway 2.8 -- only Inter-Sync Gateway Replication version 1 is available
-* For Pre-Sync Gateway 2.8+ -- you may run Inter-Sync Gateway Replication version 2 and Inter-Sync Gateway Replication version 1 (deprecated)
+* Pre-Sync Gateway 2.8 -- only SG{nbsp}Replicate is available
+* For Pre-Sync Gateway 2.8+ -- you may run Inter-Sync Gateway Replication and SG{nbsp}Replicate (deprecated)
```
```suggestion
* For Sync Gateway 2.8+ -- you may run the version of Inter-Sync Gateway Replication introduced in 2.8 or SG{nbsp}Replicate (deprecated)
```

rajagp commented on modules/ROOT/pages/_partials/common-releasenotes.adoc:
```diff
@@ -132,13 +132,13 @@ The version of Inter-Sync{nbsp}Gateway useable depends on the Sync Gateway versi
+* Pre-Sync Gateway 2.8 -- only SG{nbsp}Replicate is available
+* For Pre-Sync Gateway 2.8+ -- you may run Inter-Sync Gateway Replication and SG{nbsp}Replicate (deprecated)
 
 SG versions compatible as a passive node as part of SGR1/SGR2 replications::
```
GENERAL: Remove use of "SGR" everywhere

rajagp commented on modules/ROOT/pages/_partials/common-releasenotes.adoc:
```diff
@@ -132,13 +132,13 @@ The version of Inter-Sync{nbsp}Gateway useable depends on the Sync Gateway versi
 
-* Inter-Sync Gateway Replication v1 can target any Inter-Sync Gateway Replication version
-* Inter-Sync Gateway Replication v2 can only target Inter-Sync Gateway Replication v2+
+* SG{nbsp}Replicate can target any Inter-Sync Gateway Replication version
```
I do not know where this shows up. Whats the context? Seems very confusion the way its worded 
Why not just have a matrix which shows versions of Sync Gateway and what inter-sync gateway replications  options exist ?

rajagp replied:
GENERAL:  Use "Inter-Sync Gateway Replication introduced in Sync Gateway 2.8" to refer to new one ..Its wordy but clearer 
(Treat "SG-Replicate" as  the version of Inter-Sync Gateway Replication protocol  pre SGW 2.8 )

ibsoln replied:
@priya Can it just be "inter-Sync Gateway replication (2.8+)" ... or where contextually appropriate have 'introduced at 2.8' as a footnote?

rajagp commented on modules/ROOT/pages/_partials/config-changes.adoc:
```diff
@@ -46,40 +46,40 @@ Also, that Inter-Sync Gateway Replication does not support replication between t
+_SG Replicate Deprecated__.
 +
-*For SGR-2:* The _target_ property is replaced by the `<<remote-property>>` property.
+*For Inter-Sync Gateway replication:* The _target_ property is replaced by the `<<remote-property>>` property.
```
General : 
```suggestion
*For Inter-Sync Gateway replication introduced in Sync Gateway 2.8 :* The _target_ property is replaced by the `<<remote-property>>` property.
```

rajagp commented on modules/ROOT/pages/_partials/incpg-icr-conflict.adoc:
```diff
@@ -181,7 +181,7 @@ remoteWins::
 ====
 =====
-// Inter-Sync Gateway Replication (v2) supports automatic conflict resolution to resolve conflicting document revisions.
+// Inter-Sync Gateway Replication supports automatic conflict resolution to resolve conflicting document revisions.
```
```suggestion
// Inter-Sync Gateway Replication introduced in Sync Gateway 2.8 supports automatic conflict resolution to resolve conflicting document revisions.
```

GENERAL: If you prefer, you can also call out upfront at top of page that unless explicitly specified, all "inter-Sync Gateway replication" in page applies to Sync Gateway v2.8 and above.

ibsoln replied:
That sounds like a plan. I like it even better than my (pre 2.8 suggestion https://couchbase.slack.com/archives/CRWMUL4H5/p1602779797063200?thread_ts=1602766709.062900&cid=CRWMUL4H5)

rajagp commented on modules/ROOT/pages/_partials/incpg-icr-initialization.adoc:
```diff
@@ -61,10 +61,10 @@ Running highlights::
 --
 * Multiple identical replicators can be initiated on a Sync Gateway node provided each has a unique `replication_Id`.
-* Inter-Sync{nbsp}Gateway Replication v1 and Inter-Sync{nbsp}Gateway Replication v2 replications can run on the same node, but you must ensure that they each have a different `replication_id`.
+* SG Replicate and Inter-Sync{nbsp}Gateway replications can run on the same node, but you must ensure that they each have a different `replication_id`.
```
```suggestion
*  inter-Sync{nbsp}Gateway replications introduced in Sync Gateway 2.8 as well as SG-Replicate can run on the same node, but you must ensure that they each have a different `replication_id`.
```

rajagp commented on modules/ROOT/pages/_partials/incpg-icr-initialization.adoc:
```diff
@@ -61,10 +61,10 @@ Running highlights::
 * The user under which replication is being run must have read and write access to the data being replicated.
 * Exponential backoff when connection lost; this can be customized using the {xref-sgw-pg-config-properties-db-rep-backoff} configuration setting.
-* Inter-Sync{nbsp}Gateway Replication v2 will continue trying to connect for 30 minutes following authentication failure (including user-invalid/doesn't exist).
+* Inter-Sync{nbsp}Gateway replications will continue trying to connect for 30 minutes following authentication failure (including user-invalid/doesn't exist).
```
```suggestion
* replications will continue trying to connect for 30 minutes following authentication failure (including user-invalid/doesn't exist).
```

rajagp commented on modules/ROOT/pages/_partials/incpg-icr-initialization.adoc:
```diff
@@ -124,7 +124,7 @@ include::partial$icr-repl-props-table-sgr2.adoc[]
 == Generic Constraints
 
-.Replicating using Inter-Sync{nbsp}Gateway Replication v2
+.Replicating using Inter-Sync{nbsp}Gateway Replication
```
```suggestion
.Replication
```
Again, we should not be treating this as a name. It like saying "Replicating using Peer-to-Peer Sync" or "Replication using CBL Sync" ...Please treat it as such throughout the doc

rajagp commented on modules/ROOT/pages/_partials/incpg-icr-initialization.adoc:
```diff
@@ -139,8 +139,8 @@ This is not enforced by the system.
 Mixing Inter-Sync Gateway Replication Versions::
-Both version 1 and version 2 can legitimately be in used at the same time, especially during transition.
-However, you should avoid initializing identical version 1 and version 2 replications.
+Both Inter-Sync Gateway and SG Replicate replications can legitimately be in use at the same time, especially during transition.
```
```suggestion
Versions of inter-Sync Gateway replications introduced in 2.8 and the previous version with SG Replicate can legitimately be in use at the same time, especially during transition.
```

rajagp commented on modules/ROOT/pages/_partials/incpg-icr-initialization.adoc:
```diff
@@ -139,8 +139,8 @@ This is not enforced by the system.
-Both version 1 and version 2 can legitimately be in used at the same time, especially during transition.
-However, you should avoid initializing identical version 1 and version 2 replications.
+Both Inter-Sync Gateway and SG Replicate replications can legitimately be in use at the same time, especially during transition.
+However, you should avoid initializing identical Inter-Sync Gateway and SG Replicate replications.
```
```suggestion
However, you should avoid initializing identical replications using version introduced in 2.8 and pre-2.8 version with SG Replicate.
```

rajagp commented on modules/ROOT/pages/_partials/pn-change-log-content.adoc:
```diff
@@ -26,12 +26,12 @@ Read More . . . {xref-sgw-pg-rest-api-metrics}
 The database property has a new sub-property `replications`, which is used to associate an Inter-Sync Gateway Replication with a 'local' database.
 
-The top level `replications` configuration property and its sub-properties is deprecated. Instead, use the `database.replication` property (as above) to define Inter-Sync Gateway Replications, unless you specifically need to use v1 type Inter-Sync{nbsp}Gateway replications.
+The top level `replications` configuration property and its sub-properties is deprecated. Instead, use the `database.replication` property (as above) to define Inter-Sync Gateway Replications, unless you specifically need to use SG{nbsp}Rplicate replications.
```
```suggestion
The top level `replications` configuration property and its sub-properties is deprecated. Instead, use the `database.replication` property (as above) to define replications, unless you specifically need to use the pre-2.8 version with SG{nbsp}Replicate.
```

rajagp commented on modules/ROOT/pages/_partials/pn-change-log-content.adoc:
```diff
@@ -63,40 +63,40 @@ See: {xref-sgw-pg-rest-api-admin}
+__SG Replicate Deprecated__.
 +
-*For SGR-2:* The _target_ property is replaced by the `<<remote-property>>` property.
+*For Inter-Sync Gateway Replication:* The _target_ property is replaced by the `<<remote-property>>` property.
```
```suggestion
*For replication introduced in Sync Gateway 2.8:* The _target_ property is replaced by the `<<remote-property>>` property.
```

rajagp reviewed (changes requested):
This needs a lot of work.  I have made suggestions on a bunch of them and then I have some GENERAL comments that must be applied to the rest for consistency. (Note that I did not review all of it because  it would make more sense for you to apply those comments and re-work this)

ibsoln committed:
Apply suggestions from code review

Will do a further sanity check on these changes and submit any further changes as a new pr.

Co-authored-by: Priya Rajagopal <rajagp@users.noreply.github.com>

ibsoln committed:
Merge branch 'release/2.8' into DOC-7523
